style.css

```diff
@@ -102,36 +96,69 @@ a {
    color: #000;
 }
 
-nav ul li {
+ul li {
    font-family: 'Open Sans', sans-serif;
    display: inline;
+   list-style-type: none;
 }
 
-p {
+ul li p {
    list-style-type: none;
+   text-decoration: none;
    overflow: hidden;
    background-color: #fff;
-   /*margin: 4rem 0;*/
    padding: 6rem;
-   width: 50rem;
-   height: 20rem;
+   width: 50%;
    font-size: 1.5rem;
    color: #000;
-   clear: right;
-   margin: 0 !important;
+   float: right;
+   border-bottom: 1px solid #dfdfdf;
+   /*margin-right: 4rem !important;*/
 }
 
-section p {
-   width: 50%;
-   margin: 0 !important;
-
+/*footer {
+   width: auto !important;
 }
 
 footer p {
-   position: relative;
    background-color: #C1CDCD;
    height: 20rem;
    padding: 2rem;
-   width: auto;
    color: #000;
+}*/
+
+.read-more-state {
+  display: none;
+}
+
+span.read-more-target {
+  opacity: 0;
+  max-height: 0;
+  font-size: 0;
+  transition: .25s ease;
+}
+
+.read-more-state:checked ~ .read-more-wrap span.read-more-target {
```

Comment:
Do you know what the `~` does? Is it really necessary in this case because it seems overly complicated.